src/azure-cli/azure/cli/command_modules/appconfig/_featuremodels.py

```diff
@@ -76,10 +77,12 @@ class FeatureFlag:
     '''
     Feature Flag schema as displayed to the user.
 
+    :ivar str name:
+        Name (ID) of the feature flag.
```

Comment:
>ID [](http://example.com/codeflow?start=14&length=2)

nit: I guess key is the ID of the feature flag?

Reply:
In the feature flag schema, feature name is the `id` of feature flag. For eg, this is the raw "value" of a key-value that represents a feature flag:
```
{
	"id": "TestFeature1",
	"description": null,
	"enabled": true,
	"conditions": {
		"client_filters": []
	}
}
```